docs/userguide/basics.md

```diff
@@ -0,0 +1,89 @@
+# Basics
+A presentation (set of slides) is represented in *Elsie* with the [`Slides`](elsie.slides.Slides)
+class. The most important parameters of the presentation is the width and height of the resulting
+PDF pages, which also affects the resulting aspect ratio. The default size if `1024x768`, which
+corresponds to aspect ratio `4:3`.
+
+Here is an example of creating a new presentation:
+```python
+import elsie
+slides = elsie.Slides(width=1024, height=768)
+```
+Once you have a `Slides` object, you can create new slides from it.
+
+## Creating slides
+You can create new slides in two ways, either using the [`new_slide`](elsie.slides.Slides.new_slide)
+method or via a [decorator](#decorator). The `new_slide` method will create a new slide, but for
+convenience, it will not return the slide itself, but its [root box](layout.md), so that you can
+use the returned object immediately for adding things to the slide. Except for some advanced usage,
+you shouldn't ever need to deal with the [`Slide`](elsie.slidecls.Slide) instance itself.
+
+```python
+slide = slides.new_slide(bg_color="blue")
+```
+
+Once you have a slide, you can add content to it, for example [text](text.md), [images](images.md),
+[shapes](shapes.md) or [source code](syntax_highlighting.md).
+
+### Decorator
+A more convenient way of creating a slide is using the [`slide`](elsie.slides.Slides.slide)
+decorator. If you apply it to a function, it will create a new slide and pass its root box as a
+parameter to the function. It will also set the name of the slide according to the name of the
+decorated function.
+
+```python
+@slides.slide()
+def slide1(slide: elsie.Box):
+    slide.box().text("Hello")
+```
+
+Both `slide` and `new_slide` have parameters that allow you to change the background color and SVG
+viewbox of the slide. You can also choose its name (see [below](#name-policy))
+or enable [debug draw mode](layout.md#debug-draw-mode).
+
+## Rendering slides
+After you have created all of your desired slides and filled them with content, you can render your
+presentation using the [`render`](elsie.slides.Slides.render) method. There are several useful
+parameters of this method:
+
+- `output`: Change the output filename of the rendered PDF (default is `slides.pdf`).
+- `slide_postprocessing`: Apply some postprocessing function to all slides. See an example
+[here](../cookbook/postprocessing.md).
+- `select_slides`: Render only a selected subset of slides.
+- `return_units`: Return a list of SVG slides without rendering them. This can be useful if you
```

Comment:
New : slides_per_page=(sx, sy): where `sx` and `sy` are integers. It groups more slides into one page; more precisely there will be grid of slides `sx` times `sy`.